src/containers/Home/newsletter/Newsletter.css

```diff
@@ -0,0 +1,30 @@
+.newsletter button {
+  padding: 0.6rem;
+  outline: none;
+  border-radius: 0px 20px 20px 0px;
+  border: 2px solid #4527a0;
+  background-color: #4527a0;
+  color: white;
+}
+.newsletter success {
+  color: blue;
+}
+.newsletter input {
+  padding: 0.6rem;
+  border-radius: 20px 0px 0px 20px;
+  border: 2px solid #4527a0;
+  outline: none;
+  width: 400px;
+}
+.newsletter p {
+  color: black !important;
```

Comment:
is the `!important` necessary?

Reply:
I tried removing the !important !! it did not override the styles I want !!